### ISSUE TITLE
Pull Fields from a Related Content Type, Taxonomy, or User using templates

### ISSUE THREAD

**Issue by ghost** [pods-framework/pods#811]
Asked on the forums, but didn't get an answer. I don't THINK this is a bug, but anyway...

http://podsframework.org/forums/topic/how-do-i-pull-related-term-information-using-templates/

I'm not sure if this is my own failing to understand Wordpress, Pods, or what, but I have not been able to use a relationship field value to pull a field from the related content type, taxonomy, pod, or user. Is there a standard way to do this in a pods template?

In short, I'm trying to use the {@rel_title} field, a relationship to custom taxonomy 'Title', to pull the description field of that taxonomy. Likewise, I would like to use the {@author} relationship field sometimes to pull the bio field.
